commons/src/parsing.rs

```diff
@@ -0,0 +1,20 @@
+//! Contains the [`Element`] trait every Unimarkup element must implement.
+
+use crate::lexer::{position::Position, span::Span};
+
+/// Every Unimarkup element must implement this trait.
+pub trait Element {
+    /// Convert the element into its original plain markup form.
+    fn to_plain_string(&self) -> String;
```

Comment:
Maybe better name: `to_unimarkup` 🤔

Reply:
I will change it to `to_orig_string`.
`to_unimarkup` is not ideal, because I want the Unimarkuo output to format the content according to Unimarkup rules. e.g. combine contiguous blank lines and whitespace where possible, or add closing keywords on implicitly closed elements